proposals/simd/BinarySIMD.md

```diff
@@ -30,198 +30,210 @@ In the description below, `ImmLaneIdx{I}` indicates the maximum value of the byt
+| `f32x4.trunc`               |    `0xda`| -                  |
+| `f32x4.nearest`             |    `0xdb`| -                  |
+| `f64x2.ceil`                |    `0xdc`| -                  |
+| `f64x2.floor`               |    `0xdd`| -                  |
+| `f64x2.trunc`               |    `0xde`| -                  |
+| `f64x2.nearest`             |    `0xdf`| -                  |
+| `f32x4.abs`                 |    `0xe0`| -                  |
+| `f32x4.neg`                 |    `0xe1`| -                  |
+| `f32x4.sqrt`                |    `0xe3`| -                  |
+| `f32x4.add`                 |    `0xe4`| -                  |
+| `f32x4.sub`                 |    `0xe5`| -                  |
+| `f32x4.mul`                 |    `0xe6`| -                  |
+| `f32x4.div`                 |    `0xe7`| -                  |
+| `f32x4.min`                 |    `0xe8`| -                  |
+| `f32x4.max`                 |    `0xe9`| -                  |
+| `f32x4.pmin`                |    `0xea`| -                  |
+| `f32x4.pmax`                |    `0xeb`| -                  |
+| `f64x2.abs`                 |    `0xec`| -                  |
+| `f64x2.neg`                 |    `0xed`| -                  |
+| `f64x2.sqrt`                |    `0xef`| -                  |
+| `f64x2.add`                 |    `0xf0`| -                  |
+| `f64x2.sub`                 |    `0xf1`| -                  |
+| `f64x2.mul`                 |    `0xf2`| -                  |
+| `f64x2.div`                 |    `0xf3`| -                  |
+| `f64x2.min`                 |    `0xf4`| -                  |
+| `f64x2.max`                 |    `0xf5`| -                  |
+| `f64x2.pmin`                |    `0xf6`| -                  |
+| `f64x2.pmax`                |    `0xf7`| -                  |
+| `i32x4.trunc_sat_f32x4_s`   |    `0xf8`| -                  |
+| `i32x4.trunc_sat_f32x4_u`   |    `0xf9`| -                  |
+| `f32x4.convert_i32x4_s`     |    `0xfa`| -                  |
+| `f32x4.convert_i32x4_u`     |    `0xfb`| -                  |
+| `v128.load32_zero`          |    `0xfc`| -                  |
+| `v128.load64_zero`          |    `0xfd`| -                  |
+| `i16x8.extmul_low_i8x16_s`  |   `0x110`| -                  |
+| `i16x8.extmul_high_i8x16_s` |   `0x111`| -                  |
+| `i16x8.extmul_low_i8x16_u`  |   `0x112`| -                  |
+| `i16x8.extmul_high_i8x16_u` |   `0x113`| -                  |
+| `i32x4.extmul_low_i16x8_s`  |   `0x114`| -                  |
+| `i32x4.extmul_high_i16x8_s` |   `0x115`| -                  |
+| `i32x4.extmul_low_i16x8_u`  |   `0x116`| -                  |
+| `i32x4.extmul_high_i16x8_u` |   `0x117`| -                  |
+| `i64x2.extmul_low_i32x4_s`  |   `0x118`| -                  |
+| `i64x2.extmul_high_i32x4_s` |   `0x119`| -                  |
+| `i64x2.extmul_low_i32x4_u`  |   `0x11a`| -                  |
+| `i64x2.extmul_high_i32x4_u` |   `0x11b`| -                  |
```

Comment:
These opcodes don't match our current implementation. @tlively I believe LLVM and v8 are in sync right?

Reply:
It worked for me when I pulled latest emscripten and v8

Reply:
Yea emscripten and v8 is in sync, this document isn't.